python/cuml/tests/dask/test_dask_logistic_regression.py

```diff
@@ -557,30 +557,48 @@ def test_elasticnet(
         ("elasticnet", 2.0, 0.2),
     ],
 )
-@pytest.mark.parametrize("datatype", [np.float32])
+@pytest.mark.parametrize("datatype", [np.float32, np.float64])
 @pytest.mark.parametrize("delayed", [True])
```

Comment:
Is there a reason we parametrize this, but only test with one value? I would either test `False` as well or just remove the parameterization. I am aware that this might be copied from other tests.